docs/source/projects/user-interfaces.rst

```diff
@@ -8,12 +8,29 @@ performed using a wide range of programming languages.
 
 .. glossary::
 
+    `JupyterLab <https://jupyterlab.readthedocs.io/en/latest/>`__
+        Advanced Web-based application for authoring documents that combine live-code
+        with narrative text, equations and visualizations.
+        `Documentation <https://jupyterlab.readthedocs.io/en/latest/>`__ |
+        `Repo <https://github.com/jupyterlab/jupyterlab>`__
+
     `Jupyter Notebook <https://jupyter-notebook.readthedocs.io/en/latest/>`__
-        Web-based application for authoring documents that combine live-code
+        Simplified web-based application for authoring documents that combine live-code
         with narrative text, equations and visualizations.
         `Documentation <https://jupyter-notebook.readthedocs.io/en/latest/>`__ |
         `Repo <https://github.com/jupyter/notebook>`__
 
+    `Jupyter Desktop <https://github.com/jupyterlab/jupyterlab-desktop>`__
+        Desktop application based on JupyterLab for authoring documents that combine live-code
+        with narrative text, equations and visualizations.
+        `Repo <https://github.com/jupyterlab/jupyterlab-desktop>`__
+
+    `JupyterLite <https://jupyterlite.readthedocs.io/en/latest/>`__
+        Desktop application based on JupyterLab for authoring documents that combine live-code
```

Comment:
```suggestion
        Installation-free application based on JupyterLab for authoring documents that combine live-code
```

I would not call lite _desktop_. It is very much web-based, quote "runs entirely in the browser". It is also not officially part of Project Jupyter (while being developed by core Jupyter developers) but since we use in on try.jupyter.or already I think there would be also a consensus for including it on this list.

Maybe installation-free is not the best word, but neither is "portable" nor "self-contained". I dunno what captures "runs entirely in the browser" in a single word - maybe needs rephrasing.

Reply:
Yeah this was a mistake, I'm going to commit this change.